pkg/crc/cluster/clusteroperator.go

```diff
@@ -21,14 +21,18 @@ type Status struct {
 }
 
 func GetClusterOperatorStatus(ocConfig oc.Config, operator string) (*Status, error) {
-	return getStatus(ocConfig, []string{operator})
+	return getStatus(ocConfig, ignoreClusterOperators, []string{operator})
 }
 
-func GetClusterOperatorsStatus(ocConfig oc.Config) (*Status, error) {
-	return getStatus(ocConfig, []string{})
+func GetClusterOperatorsStatus(ocConfig oc.Config, monitoringEnabled bool) (*Status, error) {
+	ignoredOperators := ignoreClusterOperators
+	if !monitoringEnabled {
+		ignoredOperators = append(ignoredOperators, "monitoring")
+	}
+	return getStatus(ocConfig, ignoredOperators, []string{})
 }
 
-func getStatus(ocConfig oc.Config, selector []string) (*Status, error) {
+func getStatus(ocConfig oc.Config, filter, selector []string) (*Status, error) {
```

Comment:
maybe `ignoredOperators` rather than `filter` which is slightly more explicit?

Reply:
renamed!